test/core/formattersTests.ts

```diff
@@ -168,6 +168,27 @@ describe("Formatters", () => {
     });
   });
 
+  describe("shortScale", () => {
+    it("shortens long numbers", () => {
+      var fmtr = Plottable.Formatters.shortScale();
```

Comment:
I prefer naming it `format`. Or anything that is not abbreviated by omitting voewels